build_tools/github_actions/ci_build_cmake.sh

```diff
@@ -33,13 +33,14 @@ cmake -GNinja \
   -B"$STABLEHLO_BUILD_DIR" \
   -DLLVM_ENABLE_LLD=ON \
   -DCMAKE_BUILD_TYPE=Release \
-  -DLLVM_ENABLE_ASSERTIONS=On \
+  -DLLVM_ENABLE_ASSERTIONS=ON \
   -DMLIR_DIR="$LLVM_BUILD_DIR/lib/cmake/mlir" \
   -DCMAKE_CXX_COMPILER=clang++ \
   -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
   -DCMAKE_C_COMPILER=clang \
   -DCMAKE_C_COMPILER_LAUNCHER=ccache \
-  -DSTABLEHLO_ENABLE_STRICT_BUILD=On
+  -DSTABLEHLO_ENABLE_STRICT_BUILD=ON \
```

Comment:
Curious to know if  we getting any additional erros with this (like warning turned into errors)